scRNA-seq-advanced/01-read_filter_normalize_scRNA.Rmd

```diff
@@ -62,7 +63,9 @@ This directory usually contains three files:
 This will usually also include a bit of metadata about the features, including gene symbols (if the features are genes) and the type of data they represent (e.g., gene expression or antibody capture).
 - `matrix.mtx.gz`, The counts themselves, stored in a sparse ["Matrix Exchange" format](https://math.nist.gov/MatrixMarket/formats.html).
 
-Cell Ranger will also export these data in a single `HDF5` format file with a `.h5` extension, but we have found that using the separate matrix files as input seems to work more efficiently in R, so we prefer to start with those files.  
+Cell Ranger will also export these data in a single `HDF5` format file with a `.h5` extension, which can also be imported with the same commands we will use below.
+However, we have found that processing large `.h5` files is often _much_ less efficient in R, so so we prefer to start with the matrix files when possible.
+In particular, we would not recommend working with `.h5` files for raw data; the filtering steps we will use below can sometimes take hours when using those files as input.
 
 We will also need a table of mitochondrial genes, which we have stored in the `data/reference` directory. 
```

Comment:
Well, I'm here anyways, so I might ask for a slash? I tend to like slashes to denote _directory_ vs file in a written path.
```suggestion
We will also need a table of mitochondrial genes, which we have stored in the `data/reference/` directory. 
```